src/base.props

```diff
@@ -25,6 +25,11 @@
       <AdditionalIncludeDirectories>$(RA_IncludePath)</AdditionalIncludeDirectories>
       <DebugInformationFormat>ProgramDatabase</DebugInformationFormat>
       <WarningLevel>Level4</WarningLevel>
+      <PrecompiledHeader>Use</PrecompiledHeader>
+      <PrecompiledHeaderFile>pch.h</PrecompiledHeaderFile>
+      <ForcedIncludeFiles>pch.h</ForcedIncludeFiles>
+      <PreprocessorDefinitions>_ARCH=$(PlatformArchitecture);%(PreprocessorDefinitions)</PreprocessorDefinitions>
```

Comment:
I ended up not using `_ARCH`. I was going to put in in `ra_fwd.h` but decided it would be best saved for later. `/W4` doesn't check for correctness inside macros but `/Wall `does.

I figured it would be better than using `_WIN32`/`_WIN64`.